src/files/file-input/ByPathModal.js

```diff
@@ -6,9 +6,17 @@ import Icon from '../../icons/StrokeDecentralization'
 import TextInputModal from '../../components/text-input-modal/TextInputModal'
 
 function ByPathModal ({ t, tReady, onCancel, onSubmit, className, ...props }) {
+  const validatePath = (p) => {
+    if (!p.startsWith('/ipfs/')) {
+      p = `/ipfs/${p}`
+    }
+
+    return isIPFS.ipfsPath(p.trim())
```

Comment:
`trim` before `if` in case I paste "   /ipfs/QMhash"?

Reply:
No need, because we have:

```js
onChange={(p) => p.trimStart()}
```

Now that I'm looking at it, we don't even need that trim you pointed out.

Reply:
Less code FTW!